components/collection/CollectionCard.vue

```diff
@@ -51,9 +51,7 @@ onMounted(async () => {
   const metadata = (await processSingleMetadata(
     props.collection.metadata,
   )) as Metadata
-  image.value = sanitizeIpfsUrl(
-    metadata.image || metadata.thumbnailUri || metadata.mediaUri || '',
-  )
+  image.value = sanitizeIpfsUrl(getCollectionImage(metadata) || '')
```

Comment:
why?

Reply:
Didn't want to duplicate that code , that's the  why of the util function